addons/stock_dropshipping/tests/test_dropship.py

```diff
@@ -350,3 +350,61 @@ def test_add_dropship_product_to_subcontracted_service_po(self):
             {'product_id': subcontracted_service.id, 'product_uom_qty': 1.0, 'qty_delivered': 0.0},
             {'product_id': self.dropship_product.id, 'product_uom_qty': 0.0, 'qty_delivered': 1.0},
         ])
+
+    def test_report_stock_lot_customer_simple_delivery(self):
```

Comment:
Sorry, one last thing:
I would rather move it in a post-install test class of `stock` since it's not related to any dropshipping flow